### DISCUSSION TITLE
Possible to remove old hardware easily ?

### DISCUSSION THREAD

**Discussion by snodawg** [Rem0o/FanControl.Releases#699] [Q&A]
I've had Fan Control for a couple of weeks. And have donated. I just installed a new graphics card last night. 
Fan Control sees the new card but is also seeing the old card which has been removed. Should I just start from scratch with Fan Control and delete the folder  or can I edit a file to remove the old 1070 which is still listing when I run assisted setup. Thanks for the great software and hopefully I get an answer.

**Comment by xhul-dev** [comment]
Hello there.
I haven't tried it personally, but there's something called "Refresh sensors detection", in the upper right menu.
That sounds exactly like what you need.
If you're paranoid, you can backup the FanControl folder somewhere first.
Let us know how it went, cheers.

**Reply by snodawg** [reply]
Thanks but I tried the "Refresh sensors detection" several times. It keeps seeing the EVGA 1070 even though it's no longer in the machine. But it also sees the new 3070 just fine. Just wonder if the 1070 even when I hide it is affecting anything. If not, I'll just stop worrying about it for now.
